templates/patterns/footer/footer.ui_patterns.yml

```diff
@@ -8,3 +8,6 @@ footer:
     bottom_sections:
       type: array
       label: Sections in the footer
+    middle_sections:
+      type: array
+      label: Sections in the footer
```

Comment:
Change the labels, all have same label when they actually are bottom and middle sections.